Libraries/Lists/FlatList.js

```diff
@@ -666,6 +666,7 @@ class FlatList<ItemT> extends React.PureComponent<Props<ItemT>, void> {
           _removeClippedSubviews,
         )}
         {...this._renderer()}
+        accessibilityRole="table" // Inherit NSAccessibilityTable protocols
```

Comment:
Needs a macOS tag. Also I would put this above line 668, so that `{...this._renderer()}` can overwrite accessibilityRole if it wants to? I think that's a JS paradigm?